roles/openshift_control_plane/tasks/ensure_nodes_matching_selector.yml

```diff
@@ -1,4 +1,17 @@
 ---
+- name: Check that the sync daemonset is ready so labels will be up to date
+  oc_obj:
+    state: list
+    kind: daemonset
+    name: sync
+    namespace: openshift-node
+  register: __status_of_sync_ds
+
+- name: Set sync daemonset ready
+  set_fact:
+    sync_ds_ready: "{{ __status_of_sync_ds['results']['results'][0]['status']['numberAvailable']
+      __status_of_sync_ds['results']['results'][0]['status']['desiredNumberScheduled'] }}"
```

Comment:
missing comparison operator?

Reply:
Yes, thanks for catching that!  Will fix shortly.